src/main/java/utils/Utility.java

```diff
@@ -1,67 +1,119 @@
 package utils;
 
 public class Utility {
-    public static boolean isPalindrome(int number) {
-        StringBuilder reverseNumber = new StringBuilder();
-        String letter = String.valueOf(number);
-        for (int count = letter.length() - 1; count >= 0; count--) {
-            reverseNumber.append(letter.charAt(count));
+
+    public static class Boolean_Tests {
+        public static boolean isPalindrome(int number) {
+            StringBuilder reverseNumber = new StringBuilder();
+            String letter = String.valueOf(number);
+            for (int count = letter.length() - 1; count >= 0; count--) {
+                reverseNumber.append(letter.charAt(count));
+            }
+            //return reverseNumber.toString().equals(letter);
+            return true;
+        }
+
+        public static boolean oddOrEven(int nmb) {
+            return nmb % 2 == 0;
+        }
+
+        public static boolean isMultiple(int nmb1, int nmb2) {
+            return nmb2 % nmb1 == 0;
         }
-        //return reverseNumber.toString().equals(letter);
-        return true;
+
     }
 
+    public static class Number_Test {
 
-    public static int getDecimal(int binary) {
-        int decimal = 0;
-        int n = 0;
-        while (true) {
-            if (binary == 0) {
-                break;
-            } else {
-                int temporary = binary % 10;
-                decimal += temporary * Math.pow(2, n);
-                binary = binary / 10;
-                n++;
+        public static int getDecimal(int binary) {
+            int decimal = 0;
+            int n = 0;
+            while (true) {
+                if (binary == 0) {
+                    break;
+                } else {
+                    int temporary = binary % 10;
+                    decimal += temporary * Math.pow(2, n);
+                    binary = binary / 10;
+                    n++;
+                }
             }
+            return decimal;
+        }
+
+        public static double circleArea(double radius) {
+            final double PI = 3.1415926535;
+
+            return 2 * PI * radius;
         }
-        return decimal;
+
     }
 
+    public static class QuotientsAndDigits {
+        public static int Quotient(int nmb1, int nmb2) {
+            return nmb1 / nmb2;
+        }
 
-    public static int[] encrypt(int number) {
-        int[] digit = new int[4];
-        for (int i = 0; i <= 3; i++) {
-            digit[i] = (int) (number % Math.pow(10, (i + 1)) / Math.pow(10, i));
-            digit[i] += 7;
-            digit[i] %= 10;
+        public static int Remainder(int nmb1, int nmb2) {
```

Comment:
rename the methods in QuotientsAndDigits so that they follow the conventions

Reply:
fixed upercase